src/velocity.py

```diff
@@ -0,0 +1,216 @@
+import numpy as np
```

Comment:
Vamos a poner el warning bien chulo jejeje

```suggestion
import numpy as np
import warnings
```